agent/agent.go

```diff
@@ -830,20 +836,32 @@ func (a *agent) runShutdownScript(ctx context.Context, script string) error {
 	return a.runScript(ctx, "shutdown", script)
 }
 
-func (a *agent) runScript(ctx context.Context, lifecycle, script string) error {
+func (a *agent) runScript(ctx context.Context, lifecycle, script string) (err error) {
+	logger := a.logger.With(slog.F("lifecycle", lifecycle))
+
 	if script == "" {
+		logger.Info(ctx, "not running empty script")
```

Comment:
nit: isn't it more like a debug message?

PS. I'm picky as I'm working on https://github.com/coder/coder/issues/6726

Reply:
Fair. Personally I think it's good that the logs acknowledge there's no startup/shutdown script to run, and that's why it doesn't happen. Otherwise reading the logs one could wonder, why didn't it run? Open to changing the wording, and I could be wrong, maybe it's useless in which case we can turn it to debug.

Reply:
I'd rather say that it's useless here 🤔 We can deduce from the template that there is no startup script defined.